### ISSUE TITLE
Receipt notifications

### ISSUE THREAD

**Issue by IngCr3at1on** [Airbitz/airbitz-android-gui#5]
Just wondering if there's any chance at getting push notifications implemented for receiving BTC?

**Comment by swansontec** [comment]
Yes, but it will take a while. Our wallet has zero knowledge until you log in, meaning we don't know your bitcoin addresses, transaction history, exchange-rate preference, or anything else except your username. To implement a feature like this, we need access to at least some of this information while the user is logged out.

We are thinking of storing a second copy of wallet public keys unencrypted on this phone. The data on the server would remain fully encrypted, of course. That way, if somebody steals your phone, they can learn your list of bitcoin addresses and nothing more. This would allow the the notification process to watch addresses for incoming funds without requiring you to be logged in.

Given the nature of this change, it's not really a priority for us. We intend to do it eventually, but probably after multisig.

**Comment by IngCr3at1on** [comment]
Cool, makes sense. Thanks for the reply
